src/actions/index.js

```diff
@@ -43,28 +38,26 @@ export const login = values => (dispatch) => {
     });
 };
 
-export const rsvpResponse = (user, form, push) => (dispatch) => {
-  // rsvpRef
-  //   .doc(user.uid)
-  //   .set(form)
-  //   .then(() => {
-  //     dispatch({ type: types.UPDATE_RSVP, rsvpVal: true });
-  //     push('/dashboard');
-  //   });
+export const rsvpResponse = () => (dispatch) => {
+  dispatch({ type: types.UPDATE_RSVP, rsvpVal: true });
 };
 
-export const forgotPassword = (values, callback) => (dispatch) => {
+export const forgotPassword = values => (dispatch) => {
```

Comment:
Come up with a different name than `values` or remove the unnecessary `values` object and keep it as `email` for clarity.

This way, we can remove `email: values.email` in favor of just `email` in the post request.